branchVal.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+local_branch=${1}
+
+valid_branch_dash="([a-z0-9\-\]+)"
+
+
+message="There is something wrong with your branch name. Branch names in this project must adhere to this contract: $valid_branch_regex, $local_branch. Your commit will be rejected. You should rename your branch to a valid name and try again."
+
+if [[ ! $local_branch =~ $valid_branch_regex ]]
```

Comment:
The if statement should also check total length....
See the second part of the if here:  https://github.com/CMSgov/macpro-quickstart-serverless/blob/master/.github/workflows/deploy.yml#L23